resources/nheko.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Vitaly Zaitsev <vitaly@easycoding.org> -->
+<component type="desktop">
+  <id>nheko.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPLv3+ and Public Domain and MIT and Boost and Apache License and CC-BY-4.0</project_license>
```

Comment:
Not sure but shouldn't here be just the license of nheko and not of all of its dependencies?

Reply:
License of main project and all it's dependencies.